modules/delegates.js

```diff
@@ -583,10 +669,7 @@ Delegates.prototype.toggleForgingStatus = function(publicKey, secretKey, cb) {
 
 	if (encryptedItem) {
 		try {
-			decryptedSecret = __private.decryptSecret(
-				encryptedItem.encryptedSecret,
-				secretKey
-			);
+			decryptedSecret = __private.decryptSecret(encryptedItem, secretKey);
```

Comment:
Same here, it looks more readable code if you pass explicitly `encryptedItem.encryptedSecret`